docs/source/faq/index.rst

```diff
@@ -0,0 +1,45 @@
+FAQ
+===
+
+I'm new. Where to start?
+------------------------
+
+Hello, newcomer! You are very welcome :) \
+There are 2 ways for you to start with Iroha: \
+1. You can see what Iroha is an how it works by building a simple example network following our `Getting Started Guide <https://iroha.readthedocs.io/en/latest/getting_started/index.html>`_ \
```

Comment:
Add a blank line above - that will enable list formatting: 

![image](https://user-images.githubusercontent.com/36847043/47715625-55845e00-dc51-11e8-9dd6-788b81bdef29.png)